lib/gc/Analysis/MatmulConfigAnalysis.cpp

```diff
@@ -278,6 +281,13 @@ prepareConfigCandidates(Operation *root, CPUTargetDescriptionAnalysis &sysDesc,
           : getCandidate((uint32_t)shape[2],
                          shape[2] >= noSmallBlockNeedThreshold ? 8U : 1U, 256U);
 
+  if (allowIndivisibleInnerblock) {
+    innerMostKBlockCandidates = {16, 32, 64};
+    innerMostNBlockCandidates = {16, 32, 64};
+    NBlockCandidates = innerMostNBlockCandidates;
+    KBlockCandidates = innerMostKBlockCandidates;
```

Comment:
So after the change here, innermost Kblock will only be one of 16/32/64 if `allowIndivisibleInnerblock` is true?

Reply:
Yes, your understanding is correct